src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -214,7 +215,7 @@ type WebAppConfig =
     member this.ResourceId = sites.resourceId this.Name
     interface IBuilder with
         member this.ResourceId = this.ResourceId
-        member this.BuildResources location = [
+        member this.BuildResources resourceLocation = [
```

Comment:
maybe best to leave this as `location` to minimize the diff?

Reply:
Had an error happening with this one, in line 497 the webapp builder calls the ResourceGroupBuilder which requires the location property so it ends up like this which causes an error:
`location location`

Reply:
if we use a local variable to alias 'let resourceLocation = location' and use the new variable on just that one line it should solve the problem and keep the diff small (I'm worried about diverging from compositionalIT/farmer).

Reply:
ah, good point 🤦‍♂️ will do